bin/GameIndex.dbf

```diff
@@ -43673,3 +43673,1123 @@ Serial = TCPS-10172
 Name   = Homura [Taito the Best]
 Region = NTSC-J
 ---------------------------------------------
+Serial = SLKA-25080
+Name   = .hack//Kansen Kakudai Vol. 1
+Region = NTSC-K
+---------------------------------------------
+Serial = SLKA-25138
+Name   = .hack//Akushou Heni Vol. 2
+Region = NTSC-K
+---------------------------------------------
+Serial = SLKA-25145
+Name   = .hack//Shinshoku Osen Vol. 3
+Region = NTSC-K
+---------------------------------------------
+Serial = SLKA-25174
+Name   = .hack//Zettai Houi Vol. 4
+Region = NTSC-K
+---------------------------------------------
+Serial = SLKA-15023
+Name   = Assault Suits Valken
+Region = NTSC-K
+---------------------------------------------
+Serial = SLKA-25010
+Name   = Chou Battle Houshin
+Region = NTSC-K
+---------------------------------------------
+Serial = SLPM-64551
+Name   = Shin Contra
+Region = NTSC-K
+---------------------------------------------
+Serial = SLPM-64513
+Name   = Crash Bandicoot: Return of The Demon King
+Region = NTSC-K
+---------------------------------------------
+Serial = SLKA-25155
+Name   = Crimson Sea 2
+Region = NTSC-K
+---------------------------------------------
+Serial = SLPM-64532
+Name   = Digital Holmes
+Region = NTSC-K
+---------------------------------------------
+Serial = SCKA-20068
```

Comment:
Ditto.
Check the rest as well, don't wanna add more spam comments for the same issue.